### ISSUE TITLE
Nvda doesn't announce rest of text if text is too long with role=textbox

### ISSUE THREAD

**Issue by hongquan09dth5** [nvaccess/nvda#14440]
Hi Team, hope you are well. I have an div element that has an aria-readonly= true and rolex=textbox, there is a text that has three lines. I noticed when I tabbed to div element,  the NVDA just only a part of the text inside the div and ignore the rest.  If I used the mouse pointer to focus the text, it would read out the first line or second line or full of text depending on the place where I focused. Here is my example:

`
        <label id="myLabel"> viewing as: </label>
        <div role="textbox"  aria-labelledby="myLabel" tabindex="0" aria-readonly="true">is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.</div>
    `

NVDA would read out:**is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and** and ignore the rest.

NDVA version: 2022.3.2
Microsoft browser Edge version: 108.0.1462.46 (Official build) (64-bit)

**Comment by Adriani90** [comment]
This could affect performance significantly if the textbox contains a lot of text and text of different formating, tables, code syntax etc. Reading all this when tabbing into the field wouldn't make sense anyway. The current behavior gives already enough information to decide whether the text in the text box is relevant for further exploration or not.
For further exploration there are lots of further commands:
* Using arrow keys or ctr+arrow keys,
* Page up and down key or ctrl+page up and down,
* Say all command (nvda+down arrow or nvda+a)
* Object navigation and review cursor.

This being said and becauuse we don't really have  a use case why the whole text should be announced when tabbing despite all the exploring commands, I am closing as won't fix.
If you still have any arguments why this should be revisited, please comment and we can reopen.
